telemetry/metrics.go

```diff
@@ -70,61 +33,85 @@ type GatherResponse struct {
 	ContentType string
 }
 
-// New creates a new instance of Metrics
-func New(cfg Config) (_ *Metrics, rerr error) {
```

Comment:
This is API breaking for current invocations of `telemetry/New`.  If we choose to use Options instead of a static config struct the API break should be in the Changelog.

Reply:
So shall I change the pattern to config structure?

Reply:
resolved